event_model/__init__.py

```diff
@@ -120,14 +120,250 @@ def bulk_datum(self, doc):
         self.datum_page(bulk_datum_to_datum_page(doc))
 
 
+# In total wait about 2 seconds before giving up.
+# Max sleep, between the final two attempts, is about 1 second.
+DEFAULT_RETRY_INTERVALS = tuple(0.001 * numpy.array([2**i for i in range(11)]))
```

Comment:
I see why this is here and that it is too long to put directly in the signature, but this makes me a little sad (it is still not clear what the default retry behavior is from just looking at the signature).  I was thinking like `(.1, .5, 1)`

Reply:
Reasonable. Done.